erpnext_documentation/www/docs/user/manual/en/healthcare/therapy_type.md

```diff
@@ -0,0 +1,77 @@
+<!-- add-breadcrumbs -->
+
+# Therapy Type
+
+As therapists, you might be prescribing a number of different therapies for a patient right from Intensive Upper Limb Training to Functional Mobility and so on. Every therapy might have a set of exercises. ERPNext helps you templatize the therapies using the "Therapy Type" DocType. With this you can create and link therapies to Healthcare Service Units, add standard rates and Item details for billing and also add exercises according to Body Parts.
+
+To create an Exercise Type, go to:
+
+> Home > Healthcare > Rehabilitation and Physiotherapy > Therapy Type
+
+## 1. How to Create a Therapy Type
+
+1. Go to Therapy Type list, click on New.
+2. Enter a unique Therapy Name.
+3. Enter the Item details such as Item Code, Item Name, Item Group and optionally some description.
+4. Check "Is Billable" if this Therapy Type will be billed and enter the rate for one therapy session.
+5. You can optionally add the default duration for one therapy session which will be used while booking appointment for it.
+6. You can also add Healthcare Service Unit and Medical Department for the Therapy Type.
+7. Save.
+
+<img class="screenshot" alt="ERPNext Healthcare" src="{{docs_base_url}}/assets/img/healthcare/therapy-type.png">
+
+## 2. Features
+
+### 2.1 Add Exercises According to Body Parts
+
+Many a times you might need to prescribe exercises according to body parts with ailments for Patients. You can just add all the Body Parts in the _Therapy For_ field and click on "Add Exercises" to add the Exercise Types for those body parts. For that you will have to link Exercise Type to Body Parts in Exercise Type document.
```

Comment:
Many a times, you might need to advice exercises that are specific to affected body parts for Patients. You can easily add these Body Parts in the _Therapy For_ field and click on "Add Exercises" to add the Exercise Types for those body parts. For that, you will have to link Exercise Type to Body Parts in the Exercise Type document.